python/tk_maya/menu_generation.py

```diff
@@ -294,6 +294,20 @@ def add_command_to_menu(self, menu):
             
         pm.menuItem(**params)
         
+    def __exception_trap_callback_wrapper(self, callback):
+        """
+        Wrapper which traps exceptions. We execute menu commands
+        in maya via evaldeferred, meaning that all errors are just
+        swallowed by maya never to resurface. This wrapper prints 
+        them out.
+        """
+        try:
+            callback()
+        except Exception, e:
+            current_engine = tank.platform.current_engine()
```

Comment:
Why could the current engine not be set? A comment would be appreciated. Maybe also having an else that prints it out at least?